server/game/cards/03-WC/FinchCloak.js

```diff
@@ -18,7 +18,12 @@ class FinchCloak extends Card {
                         amount: 1
                     }))
                 ])
-            }))
+            })),
+            effect: '{1}',
+            effectArgs: (context) =>
+                context.player.amber < context.player.opponent.amber
+                    ? 'steal 1 amber'
+                    : 'gain both players 1 amber'
```

Comment:
What does this message resolve to? Seems like it's missing a "make"

Reply:
player1 uses Finch Cloak to gain both players 1 amber

Looks like Shakesperean english?

I can write 'to make both players gain 1 amber'

Reply:
Yep that works better :)